screens/pokedex/pokedex.gui_script

```diff
@@ -79,7 +80,7 @@ local function update_dex_data(self)
 	self.pokemon_rows = table_of_rows(self)
 end
 
-local function update_items(item)
+local function update_item(list, item)
```

Comment:
When I found "update_items" I changed it to "update_item" to be less confusing - from the list's perspective, there is only 1 item here.